### ISSUE TITLE
Checkout block: Return to Cart button is not visible in dark themes and is not RTL-friendly

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#8288]
## To reproduce

1. Select a dark theme. You can choose TT3 and go to Appearance > Editor > Styles > Browse Styles and select one of the dark variants (ie: Pilgrimage).
2. Go to Settings > General > Site Language and select a RTL language.
3. In the frontend, go to the Checkout page (with the Checkout block), making sure you have some products in the cart.
4. Notice the Return to Cart arrow is barely visible because it's black with a dark background and it points in the wrong direction in RTL languages.

![imatge](https://user-images.githubusercontent.com/3616980/214526467-59a07c2d-d77f-43e7-b32e-5e71d2434235.png)

## Expected behavior

IMO the arrow should inherit the text color. Besides that, it should point to the right in RTL languages.
